Makefile

```
@@ -23,3 +23,17 @@ tidy:
 
 version:
 	@echo $(VERSION)
+
+.PHONY: hooks
+hooks:
+	@git config core.hooksPath .githooks
+	@chmod +x .githooks/pre-commit
+	@echo "Git hooks enabled (pre-commit markdownlint autofix)."
+
+.PHONY: mdlint
```

checkmake (0.2.2)

[warning] Missing required phony target "clean"

(minphony)